### ISSUE TITLE
Homepage - find treatment validation missing from empty submission

### ISSUE THREAD

**Issue by iamjolly** [18F/samhsa-prototype#307]
### Problem Description
The find treatment form does nothing when a user tries to submit the form with a blank/empty search field. No validations seem to happen and no error message is displayed. This can be a confusing experience, especially for those with cognitive and sight-related disabilities. 

![samhsa-locate-submit-press-without-feedback](https://user-images.githubusercontent.com/1093423/64804025-49dd1580-d54b-11e9-8c26-faf67da3fb72.png)

### Recommended Solution(s)
Provide the same/similar error validation and notification experience as for those who enter invalid data in the field. See screenshot below of how this notification should also be altered for a better user experience, particularly for those with disabilities (will file related, separate issue, too).

![samhsa-locate-validation-msg](https://user-images.githubusercontent.com/1093423/64804245-b0623380-d54b-11e9-811b-184b7e34cf84.png)

Alternately, the submit button could be `disabled` via HTML attribute until valid (or any) data is in the search `input` field.
